grudge/models/euler.py

```diff
@@ -0,0 +1,364 @@
+"""Grudge operators modeling compressible, inviscid flows (Euler)"""
```

Comment:
Sprinkle some type annotations? Doesn't have to be comprehensive.

Reply:
Sprinkled in https://github.com/inducer/grudge/pull/196/commits/2f12b1edb331bdef86e76dfb45b30208ec4086ba